libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -308,8 +308,35 @@ impl ShutdownHandle {
 }
 
 #[derive(Clone, Debug)]
-struct ConnectionMetadata {
```

Comment:
The commit message should be updated to not include mention of heartbeats anymore since, both inbound and outbound connections will receive heartbeats

Reply:
Fixed